### ISSUE TITLE
ForbiddenPackageUsageCheck error

### ISSUE THREAD

**Issue by tmrobert8** [openhab/static-code-analysis#353]
There is definitely something wrong with the ForbiddenPackageUsageCheck check.  Seems like it get's 'stuck' marking the error on every class following the one that created it.

Here's an example of the incorrect check (the constants class of course only has constants and none of common collect classes)
```

org.openhab.binding.sony.internal.SonyBindingConstants.java
--

checkstyle | 2 | 22 | style | ForbiddenPackageUsageCheck | The package com.google.common.collect.BiMap should not be used.
checkstyle | 2 | 23 | style | ForbiddenPackageUsageCheck | The package com.google.common.collect.HashBiMap should not be used.
checkstyle | 2 | 49 | style | ForbiddenPackageUsageCheck | The package com.google.common.cache.Cache should not be used.
checkstyle | 2 | 50 | style | ForbiddenPackageUsageCheck | The package com.google.common.cache.CacheBuilder should not be used.
checkstyle | 2 | 51 | style | ForbiddenPackageUsageCheck | The package com.google.common.cache.RemovalListener should not be used.
checkstyle | 2 | 52 | style | ForbiddenPackageUsageCheck | The package com.google.common.cache.RemovalNotification should not be used.

```

**Comment by martinvw** [comment]
Indeed, I suppose that this method:

https://github.com/openhab/static-code-analysis/blob/6e631de725f58f4fd2b6fa5fc98bd73b38b7ec42/custom-checks/checkstyle/src/main/java/org/openhab/tools/analysis/checkstyle/ForbiddenPackageUsageCheck.java#L79-L84

requires also some reset of the `importsToLineNumbers` in the end.

**Comment by martinvw** [comment]
If anyone want to pick this up, he/she is more than welcome. Its relatively easy and isolated and the code has some existing checks but of course misses one for this case.

**Comment by wborn** [comment]
Fixed by #362
